portfolio/src/main/webapp/aboutMe.html

```diff
@@ -10,28 +10,28 @@
       <h1>About Me</h1>
       <p>I am Isabelle Dittamo!</p>
       <img src="images/profilePicture.jpg" height="300"/>
-      <p>I am an intern at Google as a part of the STEP Internship for the Summer of 2020!</p>
+      <p><br>I am an intern at Google as a part of the STEP Internship for the Summer of 2020!</p>
       <img src="images/googleGLogo.jpg" width="200"/>
       <p>
-        I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science
+        <br>I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science
         with concentrations in Information Internetworks and Intelligence.
       </p>
       <img src="images/gtLogoBuzz.png" width="200"/>
       <p>
-        At school, I am a Teaching Assistant in the College of Computing! I have previously TA'ed for the
+        <br>At school, I am a Teaching Assistant in the College of Computing! I have previously TA'ed for the
         Introduction to Computing course.
       </p>
       <img src="images/gt_coc.png" width="500"/>
       <p>
-        Outside of school, I'm a huge fan of musicals! Some of my favorites include Wicked, Les Misérables,
+        <br>Outside of school, I'm a huge fan of musicals! Some of my favorites include Wicked, Les Misérables,
         and Mamma Mia.
       </p>
       <img src="images/wickedCard.jpg" height="175"/> <img src="images/LesMiserables.png" height="175"/>
-      <p>I also love traveling around the world! Here's some snapshots of some of my previous travels around Europe.</p>
+      <p><br>I also love traveling around the world! Here's some snapshots of some of my previous travels around Europe.</p>
       <img src="images/spainJumpingPic.JPG" height="225"/> <img src="images/buckinghamPalace.JPG" height="225"/>
       <img src="images/gretnaGreenHands.JPG" width="200"/> <img src="images/scotlandCastlePic.JPG" width="200"/>
       <img src="images/irelandCastlePic.JPG" width="200"/>
-      <p>Most of all though, I love my two dogs Gumbo and Rocky!</p>
+      <p><br>Most of all though, I love my two dogs Gumbo and Rocky!</p>
```

Comment:
If the goal here is to add more vertical spacing between paragraphs, that's best done with CSS (keeps presentation separate from structure), e.g.

```css
p {
  margin-bottom: 2em;
}
```

Reply:
That makes sense! I'll do that instead.